Specs/DateTime/German/DateTimeModel.json

```diff
@@ -548,7 +542,7 @@
       "ReferenceDateTime": "2016-11-07T16:12:00"
     },
     "Comment": "Resolution is null",
-    "NotSupported": "dotnet, javascript, python",
+    "NotSupported": "dotNet, javascript, python",
```

Comment:
Any insights on why the resolution is still null?
Nit: Revert to lowercase.

Reply:
Currently all cases with "weekday"+"timeofday" in one sentence return null. Do you seen an easy way to map "weekday"+"timeofday" to "weekday"+ " "+"timeofday"? Do you think adding an optional weekdaygroup to the timeofday regex would work? I will try this some time this week.